File: JCleanArchitectureApp/appdagger/src/main/java/com/woowrale/jcleanarchitecture/data/remote/mappers/MapperToContacts.java
package com.woowrale.jcleanarchitecture.data.remote.mappers;

import com.woowrale.domain.model.Contact;
import com.woowrale.jcleanarchitecture.data.remote.model.ContactResponse;

import java.util.ArrayList;
import java.util.List;

public class MapperToContacts {

    public List<Contact> mappertoContactList(List<ContactResponse> contacts){
        List<Contact> contactList = new ArrayList<Contact>();
        if(contacts != null){
            for (ContactResponse cr: contacts) {
                contactList.add(new Contact(cr.getName(), cr.getProfileImage(), cr.getPhone(), cr.getEmail()));
            }
        }
        return contactList;
    }
}

File: KCleanArchitectureApp/presentation/src/main/java/com/woowrale/kcleanarchitecture/data/remote/mappers/MapperToContacts.kt
package com.woowrale.kcleanarchitecture.data.remote.mappers

import com.woowrale.domain.model.Contact
import com.woowrale.kcleanarchitecture.data.remote.model.ContactResponse
import java.util.*
import kotlin.collections.ArrayList

class MapperToContacts {
    fun mappertoContactList(contacts: List<ContactResponse>): List<Contact> {
        val contactList: MutableList<Contact> = ArrayList<Contact>()
        if (contacts != null) {
            for (cr in contacts) {
                contactList.add(Contact(cr.name, cr.profileImage, cr.phone, cr.email))
            }
        }
        return contactList
    }
}